cli/src/main/java/io/specmesh/cli/Provision.java

```diff
@@ -146,6 +140,22 @@ public void spec(final String path) {
         builder.specPath(path);
     }
 
+    @Option(
```

Comment:
new command line option for specifying an explicit alternative user to use when creating acls.